### ISSUE TITLE
 Too few items for physical damage?

### ISSUE THREAD

**Issue by VetitatiS** [RobertSkalko/Mine-and-Slash#275]
I play with friends as a swordsman class without spells, but most of the unique items and amulets provide mostly magical parameters. Maybe I don't understand everything, but in my opinion it's worth adding unique items for players who want to play without magic damage. Of course, there are items like Headhunter, but this is more of an exception.

**Comment by RobertSkalko** [comment]
Yes the plan is to add many more unique items for all build types. As they are made when I get inspiration + have assets ready, it means some builds will always have more choices.
